base/libgit2/types.jl

```diff
@@ -449,6 +449,16 @@ struct FetchHead
     ismerge::Bool
 end
 
+function Base.show(io::IO, fh::FetchHead)
+    println(io, "FetchHead:")
+    println(io, "Name: $(fh.name)")
+    println(io, "URL: $(fh.url)")
+    print(io, "OID: ")
+    show(io, fh.oid)
```

Comment:
Can this line be combined with the ones above and below it, i.e. `println(io, "OID: ", fh.oid)`?